src/LintingViewExtension/Controls/IRuleIssue.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Dynamo.Linting.Rules;
+
+namespace Dynamo.LintingViewExtension
+{
+    public interface IRuleIssue
```

Comment:
I taught it will be enough to rely solely on rule evaluation results that are being provided by the Linter manager. We can enhance those results if needed. I was hoping the view would only need to be aware of the linter manager and its results to do its job. Do we still need IRuleIssue when we have IRuleEvaluationResults ?

Reply:
It is still only aware of the LinterManager and its results. The RuleIssues are essentially wrapper models that make it easier for me to create the UI bindings. In the UI i need to display multiple results under the same issue group, so every time a result comes in I check if the result belongs to an issue section that is already created, if it does i simply add the affected nodes to that section and if it doesn't i create a new one. 

Im not sure how easy it would be to rely on the EvaluationResults for the UI, in my mind there would need to be something to summarize them all and make sure they are grouped together correctly in the UI.

The interface is simple to hide the two concrete implementations `NodeRuleIssue` and `GraphRuleIssue`, so instead of making both of them public i can use the public interface (As im binding the UI to a colleciton of these they will need to be public, otherwise binding dosent work).